src/Constants/Message.js

```diff
@@ -0,0 +1,25 @@
+export const SYSTEM_MESSAGE = Object.freeze({
+  INPUT_PURCHASE_AMOUNT: '구입금액을 입력해 주세요.\n',
```

Comment:
에어비앤비 컨벤션에 따르면 [상수 객체의 키는 대문자 대신 camelCase를 권장](https://github.com/airbnb/javascript#naming--uppercase)한다고 합니다!
```suggestion
inputPurchaseAmount: '구입금액을 입력해 주세요.\n',
```